LivlReviews.Domain.Test/Stubs/StubRequestInventory.cs

```diff
@@ -1,17 +1,40 @@
 using LivlReviews.Domain.Domain_interfaces_output;
 using LivlReviews.Domain.Entities;
+using LivlReviews.Domain.Enums;
 
 namespace LivlReviews.Domain.Test.Stubs;
 
-public class StubRequestInventory(ProductStock stock) : IRequestInventory
+public class StubRequestInventory(List<ProductStock> stocks, List<Request> requests) : IRequestInventory
```

Comment:
https://knplabs.com/fr/blog/mocks-fakes-stubs-dummy-et-spy-faire-la-difference/

In reality this is not a Stub, but a Fake I think (refering to this article).
I have also missnamed MockInvitationTokenInventory that is a Fake too.

Reply:
☝️ 🤓